### PR TITLE
Disable alerts

### PR DESCRIPTION
**Pull request by tzachi-dar**
Import of the code that disables alerts.

Since it can be deleted, it brings back the 55 undeletable alert. We might consider allowing to delete it.

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/AlertList.java [hunk 1]
```diff
@@ -159,9 +173,36 @@ public void onClick(View v) {
...
     }
 
+    void displayWarning() {
+        runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+
+                if (!isFinishing()){
+                    new AlertDialog.Builder(AlertList.this)
+                      .setTitle("Warnining !")
```
Comment (jamorham) [inline app/src/main/java/com/eveningoutpost/dexdrip/AlertList.java]: Can we change the title to: "Warning !"

File: app/src/main/java/com/eveningoutpost/dexdrip/AlertList.java [hunk 2]
```diff
@@ -159,9 +173,36 @@ public void onClick(View v) {
...
+                      .setMessage("No active low alert exists, it is highly recomended that you will add a low alert.")
```
Comment (jamorham) [inline app/src/main/java/com/eveningoutpost/dexdrip/AlertList.java]: Can we change the message to: "No active Low Alert exists, without this there will be no alert on low glucose! Please add or enable a low alert."

File: app/src/main/java/com/eveningoutpost/dexdrip/Models/AlertType.java [hunk 3]
```diff
@@ -361,10 +358,10 @@ public static void update_alert(
         at.save();
     }
     public static void remove_alert(String uuid) {
-  /*      if(uuid.equals(LOW_ALERT_55)) {
+        if(uuid.equals(LOW_ALERT_55)) {
```
Comment (jamorham) [inline app/src/main/java/com/eveningoutpost/dexdrip/Models/AlertType.java]: There should not be any alerts which cannot be removed/disabled if the user wishes to do so. This restriction should be reverted.

**Comment by jamorham** [comment]
I've added some line comments to hopefully make it easier with my suggested changes. 

These are for wording on the warning and that we should not have an unchangeable alert because it removes flexibility and user choice and would change how the code currently works which I think is the most agreeable.

**Comment by tzachi-dar** [comment]
I have fixed the issues that you have raised.
As for testing, I have checked that the UI works.
The code is just a clone of what we had in xDrip.

I will test actual alerts tonight.

**Comment by tzachi-dar** [comment]
Verified alerts are working/not working as needed.

**Pull request merged**